### DISCUSSION TITLE
newscon.org and my filter, a blow to the save my filters to wake up ublock origin

### DISCUSSION THREAD

**Discussion by gabri25ele** [uBlockOrigin/uAssets#25837] [Q&A]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/user-attachments/assets/1879fc88-09bc-4de4-9e2f-3834a6ab396a), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://www.newscon.org/d3/
```

### Description

as mentioned above, I am using newscon.org##+js(nano-sib, , ,,0.01)
on the site in question
meanwhile there is the problem that this code makes the link unclickable
Can anyone explain why?

One upon a time there was an instant countdown, I don't know how they had done it in ublock origin automatic filters, but now you have to (re)use this newscon.org##+js(nano-sib, , ,0.01) 

but the strangest thing and the main topic of this discussion is the fact that after a certain amount of time, after a restart of the browser the next day, something, but I think it has something to do with the time passing, the countdown is slower despite the fact that the filter is always the same.
Do you know how I resolve everything?
I simply go to my filters, don't touch anything, put a space or delete it from a blank line, save, refresh the page, and the countdown goes back up to full speed,
isn't this strange?
to me it seems like the weirdest thing in the universe.
why is it necessary to tell ublock origin to do something? what does saving my filters correspond to even if no changes are made? and does anything change?
to me it is strange.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

every browser wth ublock origin

**Comment by gorhill** [comment]
> after a restart of the browser

Chromium-based browsers are not waiting for uBO to be ready at launch for already opened tabs, so your scriptlet-based filter might be injected too late. Just a reload of the page typically fixes this. See <https://github.com/gorhill/uBlock/wiki/uBlock-Origin-works-best-on-Firefox#browser-launch>.
